test/llvmDebug/llvmDebug_test.py

```diff
@@ -0,0 +1,71 @@
+# Python test script to verify the llvm debug info
+# is generated properly during Chapel compilation
```

Comment:
Oh yeah, and if you do that, this directory will need to skip if the platform is not linux64.